package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/connect-js",
-  "version": "1.0.9",
+  "version": "1.0.10",
```

Comment:
You shouldn't bump this in this PR! We should bump after the regular change is merged. The PR should also have a reasonable description/title.